feat(icons): new UI icons from August - September 2025

DanaPaslaru opened the pull request:
Submitting new UI icons from August throuh September 2025, all additions and updates approved by BXD 27 new icons updated categories.yml with additions and updates updated icons.yml with additions

2 updated icons


### Changelog

**New**

32/bsam-qsam-zedc.svg
32/classic-batch.svg
32/coupling-facility-encryption.svg
32/data-set-encryption.svg
32/dfsort-ibm-z-sort.svg
32/digital--identity.svg
32/fire--fill.svg
32/global-filters.svg
32/graph-aggregator.svg
32/ibm--webmethods-api-gateway.svg
32/ibm--webmethods-api-studio.svg
32/ibm--webmethods-b2b-integration.svg
32/ibm--webmethods-developer-portal.svg
32/ibm--webmethods-hybrid-integration.svg
32/ibm--webmethods-integration-server.svg
32/ibm--webmethods-integration.svg
32/ibm--webmethods-managed-file-transfer.svg
32/ibm-federated-api-management.svg
32/ibm-hybrid-control-plane.svg
32/logo--bluesky.svg
32/platform-automation.svg
32/Serverless-Fleet.svg
32/top-data-sets.svg
32/top-programs.svg
32/update-complete.svg
32/value--variable--alt.svg
32/z-HyperLink.svg

**Changed**

32/thumbs-up-double--filled.svg
32/thumbs-up-double.svg

DanaPaslaru committed:
feat(icons): new UI icons from August - September 2025

Submitting new UI icons from August throuh September 2025, all additions and updates approved by BXD
27 new icons updated categories.yml with additions and updates updated icons.yml with additions

2 updated icons

DanaPaslaru committed:
Update icons.yml

laurenmrice reviewed (changes requested):
@DanaPaslaru 

**graph-aggregator**
Some of the lines in this icon do not seamlessly connect to the circles.
<img width="784" height="418" alt="Screenshot 2025-10-06 at 10 31 28 AM" src="https://github.com/user-attachments/assets/b03e0b99-e7e9-4128-b913-9aa0ec242c56" />

**platform-automation**
This icon is under two different categories in the categories/yml: Systems and Technology. Was that intentional? Typically we only place it under one category so just wanted to confirm.

**z-HyperLink**
This icon does not seem to be listed in the categories.yml file.

DanaPaslaru committed:
updated pr with fixed ui icon and updates on categories.yml file

updated:
- replaced ui icon "graph-aggregator.svg" with new artwork
- removed  "platform-automation.svg" from Technology category
- listed "z-HyperLink.svg" to Systems in categories.yml file

DanaPaslaru committed:
updated categories.yml

removed "z-batch-reporter" mention from categories.yml file

ariellalgilmore committed:
chore(snapshot): update

tay1orjones reviewed (changes requested):
Running `node tasks/scaffold.js` produces changes to the metadata file

tay1orjones committed:
chore: update icons metadata

tay1orjones reviewed (approved):
I went ahead and pushed the updates - it was only modifications to single quotes where needed and updated alphabetical ordering. If you want to double check you can view the commit just above this comment.
